src/pages/plugins/plugins.js

```diff
@@ -253,7 +254,8 @@ export default function PluginsInclude(updates) {
       return await response.json();
     } catch (error) {
       $list.all.setAttribute("empty-msg", strings["error"]);
-      console.error(error);
+      window.log("error", "Failed to filter plugins:");
+      window.log("error", error);
```

Comment:
Used to do one Job Twice

Reply:
In first time sending the string log then sending the actual error object